package.json

```diff
@@ -20,6 +20,7 @@
     "@segment/snippet": "^4.4.0",
     "@segment/utm-params": "^2.0.0",
     "@sentry/node": "^4.3.0",
+    "@types/storybook-react-router": "^1.0.1",
```

Comment:
Imo `@types` should go in `devDependencies` :)

Reply:
Ah of course, my bad!